exporters/trace/zipkin/README.md

```diff
@@ -64,7 +64,7 @@ This will report Zipkin v2 json format to a single server. Alternate
 ```java
 public class MyMainClass {
   public static void main(String[] args) throws Exception {
-    ZipkinTraceExporter.createAndRegister("http://127.0.0.1:9411/api/v2/spans");
+    ZipkinTraceExporter.createAndRegister("http://127.0.0.1:9411/api/v2/spans", "myService");
```

Comment:
nit service names in zipkin end up lowercased. so 'service-a' or 'my-service' is better

Reply:
Got it, posted https://github.com/census-instrumentation/opencensus-java/pull/1038 to update.